<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>IntroWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(255, 255, 255)
}
QLabel{
	background-color: rgb(255,255,255,0);
}
QPushButton{
	border:1px solid rgb(84, 84, 85);
	background-color: rgb(255, 255, 255);
	padding: 12px;
	border-radius: 8px;
	border-bottom: 4px solid grey;
	
	color: rgb(0, 0, 0)
}</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>130</y>
      <width>451</width>
      <height>431</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/kkambback.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>770</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>여기어때 잘난체 OTF</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Start!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>540</y>
      <width>1001</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>여기어때 잘난체 OTF</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Don't worry about your eye</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>770</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>여기어때 잘난체 OTF</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Git Hub</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>790</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/kdata_og.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>780</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/korea_univ.png</pixmap>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btn_go_to_menu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1196</x>
     <y>826</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btn_github()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1391</x>
     <y>791</y>
    </hint>
    <hint type="destinationlabel">
     <x>1479</x>
     <y>669</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_go_to_menu()</slot>
  <slot>btn_github()</slot>
 </slots>
</ui>
